frontend/lib/src/components/elements/ArrowTable/ArrowTable.test.tsx

```diff
@@ -28,19 +30,23 @@ const getProps = (data: Uint8Array): TableProps => ({
 describe("st._arrow_table", () => {
   it("renders without crashing", () => {
     const props = getProps(UNICODE)
-    const wrapper = mount(<ArrowTable {...props} />)
+    render(<ArrowTable {...props} />)
 
-    expect(wrapper.find("StyledTable").length).toBe(1)
-    expect(wrapper.find("StyledTableContainer").length).toBe(1)
-    expect(wrapper.find("StyledEmptyTableCell").exists()).toBeFalsy()
+    expect(screen.getByTestId("stTable")).toBeInTheDocument()
+    expect(screen.getByTestId("stTableStyledTable")).toBeInTheDocument()
+    expect(screen.queryAllByTestId("stTableStyledEmptyTableCell").length).toBe(
+      0
+    )
   })
 
   it("renders an empty row", () => {
     const props = getProps(EMPTY)
-    const wrapper = mount(<ArrowTable {...props} />)
+    render(<ArrowTable {...props} />)
 
-    expect(wrapper.find("StyledTable").length).toBe(1)
-    expect(wrapper.find("StyledTableContainer").length).toBe(1)
-    expect(wrapper.find("StyledEmptyTableCell").exists()).toBeTruthy()
+    expect(screen.getByTestId("stTable")).toBeInTheDocument()
+    expect(screen.getByTestId("stTableStyledTable")).toBeInTheDocument()
+    expect(screen.queryAllByTestId("stTableStyledEmptyTableCell").length).toBe(
```

Comment:
Nit: this could just be:
`expect(screen.getByTestId("stTableStyledEmptyTableCell")).toBeInTheDocument()`